Chinese/SeanLau/SeanLau.md

```diff
@@ -0,0 +1 @@
+至尊宝转世为刘青云，拼命要寻找其他六位彩云，希望终有一日回去救紫霞仙子 [Married](married/married.md)
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.